test/e2e/smoke_test.go

```diff
@@ -48,7 +48,7 @@ func (s *SmokeSuite) TestWorkflowTemplateBasic() {
 		When().
 		CreateWorkflowTemplates().
 		SubmitWorkflow().
-		WaitForWorkflow(30 * time.Second).
+		WaitForWorkflow(60 * time.Second).
```

Comment:
If we're increasing the time here we should make sure we are also increasing it on the Makefile (`go test ...`)